typings/custom-ckeditor-defs.d.ts

```diff
@@ -2,6 +2,15 @@
 // https://github.com/ckeditor/ckeditor-dev/issues/2947 is resolved.
 declare namespace CKEDITOR {
     interface pluginDefinition {
+        hidpi?: boolean;
+        lang?: string | string[];
         icons?: string;
+        requires?: string | string[];
+
+        afterInit?(editor: editor): any;
+        beforeInit?(editor: editor): any;
```

Comment:
Why was this typed as any?

Reply:
because it is in ckeditor official docs

Reply:
for all of them right?

Reply:
yes

Reply:
In that case, can you put the link to the docs at the top of the top of the file so that we know where to look in the future?

Reply:
Here is the link of ckeditor docs
https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_pluginDefinition.html

Reply:
Sorry, I meant putting it into the codebase.

Reply:
@NoumanMasood have you gotten a chance to get around to this?